conf/base.config

```diff
@@ -21,6 +21,13 @@ process {
   maxRetries = 1
   maxErrors = '-1'
 
+  withName:preprocess_bam {
```

Comment:
I think this should be for the `make_examples` process as that is the most computationally intensive process, however, its probably worth doing it for all of the main deepvariant processes

Reply:
Yes, it is :-)